torch/_tensor_docs.py

```diff
@@ -3619,11 +3684,33 @@ def callable(a, b) -> number
 See :func:`torch.topk`
 """)
 
+add_docstr_all('to_dense',
+               r"""
+to_dense() -> Tensor
+
+Returns a dense copy of the :attr:`self` tensor. Here, a dense tensor
+is a strided tensor (i.e., with `torch.strided` layout).
+
+If the :attr:`self` tensor is already a strided tensor, this throws an error.
```

Comment:
"Throws an error if self is a strided tensor."

Reply:
What do you think about putting these error-throwing statements in warnings to separate them from the descriptive documentation?

Reply:
I think it is a good idea. Applied.